composer.lock

```diff
@@ -1,5790 +0,0 @@
-{
```

Comment:
`composer.lock` should be preserved: it doesn't affect downstream, but it does affect the stability of our CI environment.

Reply:
Sorry for misinterpreting the situation. It was enough to remove psr/log from composer.json, composer.lock didn't do anything of preventing anything :) I've added it back.